adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -701,6 +707,10 @@ private void waitingRequestOnError(final CallbackHandler handler, final Authenti
         }
     }
 
+    public final boolean isOutageMode() {
+        return mAuthContext.getExtendedLifetimeEnabled();
```

Comment:
this function does not seem to be called anywhere

---
In reply to: [70145107](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r70145107) [](ancestors = 70145107)